fiasko_bro/code_validator.py

```diff
@@ -126,14 +126,14 @@ def _run_warning_validators_until(self, failed_error_group_name, arguments):
             )
         return warnings
 
-    def run_validator_group(self, group, pre_validation=False, *args, **kwargs):
+    def run_validator_group(self, group, add_warnings=False, *args, **kwargs):
         errors = []
         for error_group_name, error_group in group.items():
             errors += self._run_validator_group(
                 error_group,
                 self.validator_arguments
             )
-        if pre_validation and errors:
+        if add_warnings and errors:
```

Comment:
Compared to `pre_validation`, the meaning of `add_warnings` is reverted. As a result, this code does not what it's supposed to.